CONTRIBUTING.md

```diff
@@ -5,56 +5,107 @@ When contributing to this repository, please describe the change you wish to mak
 Please note we have a [code of conduct](CODE_OF_CONDUCT.md), please follow it in all your interactions with the project.
 
 ## Setting Up Development Environment
-Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools. 
-### Prerequisites
+
+Before making any changes to the source code (documentation excluded) make sure you have installed all the required tools.
+
+### With Devbox
+
+The easiest way to setup and build devbox is with Devbox!
+
+* Install Devbox using the command below. If you don't have Nix installed, Devbox will automatically install it for you when you run a command.
+
+```bash
+curl -fsSL https://get.jetpack.io/devbox | bash
+```
+
+* Start a Devbox Shell for the project
+
+```bash
+devbox shell
+```
+
+* Build the Devbox CLI:
+
+```bash
+# Build for your current OS
+devbox run build
+
+# Build for Linux
+devbox run build-linux
+
+# Run the CI Linter
+devbox run lint
+```
+
+* Run and Test the Devbox CLI:
+
+```bash
+./dist/devbox <test_command>
```

Comment:
I think `devbox run build` prepends the binary (./dist/devbox) to $PATH so just running a devbox command would use the build binary

Reply:
Would that change perisist if you're outside of a devbox shell?

Reply:
@Lagoja right! it wouldn't because updating the path is in the init_hook. So ignore this comment. Running `dist/devbox` is a safer option.